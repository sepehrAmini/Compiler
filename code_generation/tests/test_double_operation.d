int main(){
    double a;
    double b;
    double c;
    a = 1.5;
    b = 3.4;
    c = a + b;
    Print(c);
    c = a - b;
    Print(c);
    c = a * b;
    Print(c);
    c = a / b;
    Print(c);
}