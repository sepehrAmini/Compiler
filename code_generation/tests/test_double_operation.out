4.90000010